jena-jdbc/jena-jdbc-driver-remote/src/test/java/org/apache/jena/jdbc/remote/results/TestRemoteEndpointResultsWithResultSetTypes.java

```diff
@@ -48,7 +49,7 @@
     @BeforeClass
     public static void setup() throws SQLException {
         ServerTest.allocServer();
-
+        HttpOp.setDefaultHttpClient(null);
```

Comment:
Is this necessary? If it is, could have an explaining comment?  Rekalted to AbstractRemoteEndpointResultSetTests / caching client?